modules/GenericSensor.js

```diff
@@ -126,8 +127,13 @@ function GenericSensor() {
                            sensor.z !== null,
                            "sensor: reading value for '" + sensorType + "'");
 
-                    assert(sensor.controller === "bmi160",
-                           "sensor: controller name as default");
+                    if (sensorType === "Magnetometer") {
```

Comment:
Li Hao and I have discussed this issue, so we support your point of view. I will delete this test point.